site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -494,22 +520,71 @@ being rejected.
 The AMQP adapter client can be used to simulate a device which receives commands and sends responses back to the
 application.
 
-A subscription to commands for a specific device can be done like this:
+Start the client in interactive mode:
+
+~~~sh
+# in directory: hono/cli-quarkus/target/
+java -jar hono-cli-*-exec.jar amqp -H hono.eclipseprojects.io -P 5672 -u sensor1@DEFAULT_TENANT -p hono-secret
+hono-cli/amqp-device>
+~~~
+
+{{% notice tip %}}
+When interacting with Hono's Sandbox AMQP adapter, the host, port and credentials options can be replaced by the
+single `--sandbox` option. In this case, the client will authenticate as device `4711` of the `DEFAULT_TENANT`.
+{{% /notice %}}
+
+The following command will start receiving commands for the authenticated device (`4711`) and automatically send a
+response using status code `204`:
+
+~~~sh
+# at the hono-cli/amqp-device> prompt
+sub -s 204
+~~~
+
+{{% notice tip %}}
+The `sub` command also accepts a `-d` option. This can be used by gateway devices to start receiving commands for
+one of the devices that it may act on behalf of.
+{{% /notice %}}
+
+When a command is received, the client will print a line to *System.out* consisting of:
```

Comment:
I would suggest using `print a line to the console` here (`System.out` would be Java specific).